1-sum-app/sum-app.css

```diff
@@ -0,0 +1,44 @@
+.wrapper-content{
+
+    margin: auto;
+    text-align: center;
+
+}
+.input-wrapper{
+    width: 400px;
+    height: 350px;
+    margin: auto;
+
+    text-align: center;
+    margin-top: 28px;
```

Comment:
Always better to put the put similar CSS one below other. In this case `margin: auto;` and ` margin-top: 28px;` must be once below other.